flink-tests/src/test/java/org/apache/flink/test/checkpointing/StreamCheckpointingITCase.java

```diff
@@ -53,6 +62,8 @@
 @SuppressWarnings("serial")
 public class StreamCheckpointingITCase {
 
+	private static final Logger LOG = LoggerFactory.getLogger(StreamCheckpointingITCase.class);
```

Comment:
Before I decided to undo parts of the checkpointing changes (sending the full state back  from the JM to the tasks), I had added an additional test there.

I can delete the logger factory.